### PR TITLE
Added missing model for Order resource (Payment)

### PR DESCRIPTION
**Pull request by ddoehring**
When getting a specific Order, you also get a Payment-model.
This model was missing, so I added it ;).

### CODE REVIEW

File: source/developers-guide/rest-api/models/index.md [hunk 1]
```diff
@@ -562,6 +562,30 @@ subgroup: REST API
 | group                    | string                  |                                                 |
 | sendMail              | boolean                  |                                                 |
 
+## Payment
+
+* **Model:** Shopware\Models\Customer\Payment
```
Comment (mitelg) [inline source/developers-guide/rest-api/models/index.md]: this model and this table don't exist in Shopware. therefore I will decline your PR. 

I'm not quite sure, what your intention was behind that, but feel free to open another PR :blush: :+1:

Comment (ddoehring) [reply source/developers-guide/rest-api/models/index.md]: `
### {
    "id":"",
    "name":"",
    "description":"",
    "template":"",
    "hide":"",
    "additionalDescription":"",
    "debitPercent":"",
    "surcharge":"",
    "surchargeString":"",
    "position":"",
    "active":"",
    "esdActive":"",
    "mobileInactive":"",
    "pluginId":""
}
--

So this is the JSon structure I'm getting back from calling http://sw-demo.dev.1drop.de/api/orders/2 (i left out the values in the response).

I didn't find a model, that matches this structure but in the docs, Payment is mentioned and linked to (at least it tries). So I thought it's missing, isn't it :smile:?

`

Comment (mitelg) [reply source/developers-guide/rest-api/models/index.md]: this looks like `\Shopware\Models\Payment\Payment` and the table `s_core_paymentmeans`
It's missing, you are right. 

indeed the link on this page https://developers.shopware.com/developers-guide/rest-api/api-resource-orders/ seems to be wrong. `Payment` redirects to `Payment-Instance` which is wrong.

Maybe you can change these both things? then we have the correct state :blush: :+1:

**Comment by ddoehring** [comment]
Sadly i was unable to extends this PR to include my second PR (fixing payment model link) 😃. So i made a new PR.
Both Pull Requests have no changes, which all together fixes the mentioned issues.

Sorry for this inconvinience 😃

**Comment by mitelg** [comment]
thanks @ddoehring :+1:

**Pull request merged**